packages/studio-ui/src/web/components/Content/Select/Widget.tsx

```diff
@@ -116,14 +116,19 @@ class ContentPickerWidget extends Component<Props> {
 
     return (
       <ControlGroup fill>
-        <InputGroup
-          placeholder={placeholder}
-          value={textContent}
-          disabled
-          id={inputId || ''}
-          className={style.contentInput}
-        />
-        {contentItem && <Button icon="edit" onClick={this.editItem} className={Classes.FIXED} />}
+        <div
+          style={{ display: 'flex', flexDirection: 'row'}}
```

Comment:
Please avoid style when possible, use a className and put the css in the .scss file